claim/README.md

```diff
@@ -116,6 +116,67 @@ For the connection process to work, the contents of `/var/lib/netdata` _must_ be
 restarts using a persistent volume.  See our [recommended `docker run` and Docker Compose
 examples](/packaging/docker/README.md#create-a-new-netdata-agent-container) for details.
 
+#### Known issue on systems with enabled seccomp
+
+The nodes running on the following hosts **cannot be claimed**:
+
+- `libseccomp` version less than v2.3.3.
+- Docker version less than v18.04.0-ce.
+- the kernel is configured with CONFIG_SECCOMP enabled.
+
+To check if your kernel supports `seccomp`:
+
+```cmd
+# grep CONFIG_SECCOMP= /boot/config-$(uname -r) 2>/dev/null
+CONFIG_SECCOMP=y
+```
+
+To resolve, do one of the following:
+
+- Update to a newer version of Docker and `libseccomp` (recommended).
+- Create a custom profile and pass it for the container.
+- Run [without the default seccomp profile](https://docs.docker.com/engine/security/seccomp/#run-without-the-default-seccomp-profile) (unsafe, not recommended).
+
+<details>
+<summary>See how to create a custom profile</summary>
+
+- download the moby default seccomp profile and change `defaultAction` to `SCMP_ACT_TRACE` on line 2.
+
+```cmd
+sudo wget https://raw.githubusercontent.com/moby/moby/master/profiles/seccomp/default.json -O /etc/docker/seccomp.json
+sudo sed -i '2s/SCMP_ACT_ERRNO/SCMP_ACT_TRACE/' /etc/docker/seccomp.json
+```
+
+- explicitly specify the new policy for the container.
+
+When using `docker run`:
+
+```cmd
+docker run -d --name=netdata \
+  --security-opt=seccomp=/etc/docker/seccomp.json \
+  ...
+```
+
+When using `docker-compose`:
+
+> :warning: The security_opt option is ignored when deploying a stack in swarm mode.
+
+```yaml
+version: '3'
+services:
+  netdata:
+    security_opt:
+      - seccomp:/etc/docker/seccomp.json
+    ...
+```
+
+When using `docker stack deploy`:
+
+Change the default profile globally by adding `--seccomp-profile=/etc/docker/seccomp.json` to the options passed to
+dockerd on startup.
```

Comment:
These all relate to step 2, right? 

Can you indent these lines using three spaces? Then it will be displayed as a list continuation.

Reply:
I think it won't because there is the text between. Will check.

Reply:
> These all relate to step 2, right?

Yes, should be clear from the context.

<img width="802" alt="Screenshot 2022-02-21 at 12 38 09" src="https://user-images.githubusercontent.com/22274335/154928362-9654fd24-d9a1-4a3a-b2c6-03f2403f60c8.png">